integration-tests/md/extend.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/python
+
+# requirements: md raid /dev/md0 with partitions sdb[1-4] and unused partition sdb5
+
+
+from sys import exit
+from storage import *
+from storageitu import *
+
+
+set_logger(get_logfile_logger())
+
+environment = Environment(False)
+
+storage = Storage(environment)
+
+staging = storage.get_staging()
```

Comment:
The staging devicegraph since Storage::get_staging() returns a Devicegraph. Do you prefer ```staging_devicegraph = storage.get_staging_devicegraph()```?